tests/python/unit/test_geodata.py

```diff
@@ -12,23 +13,14 @@ def convert_numpy_int64_to_int(o):
 def test_wkt_parser():
     wkt_point = "POINT(-45 59)"
     result = wkt_parser(wkt_point)
-    
-    assert False
-
-
-def test_extract_lat_long():
-    coordinates = ['POINT(-73.97237 40.64749)', 'POINT(-73.98377 40.75362)', 'POINT(-73.9419 40.80902)']
-    lat, long_ = extract_lat_long(coordinates)
-    assert False
-
-
-def test_extract_lat_long_invalid():
-    coordinates = [None, 'POINT(-73.98377 40.75362)', 'POINT(-73.9419 40.80902)']
-    lat, long_ = extract_lat_long(coordinates)
-    assert False
+    assert result[0] == '-45'
+    assert result[1] == '59'
 
 
-def test_format_geodata():
+def test_filtering_lead_to_empty_df():
```

Comment:
This test returns an error because of the `dataiku`package in `format.py`

Reply:
Deleted the Dataset typing leading to the error